mappings/servers/fara/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "fara",
+    "name": "FaraMC",
+    "description": "The next gen of Minecraft Minigames",
+    "addresses": [
+        "faramc.uk", "tcpshield.faramc.uk", "neoprotect.faramc.uk", "playit.faramc.uk"
+    ],
+    "primaryAddress": "faramc.uk",
+    "minecraftVersions": [
+        "1.8.*", "1.21.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#800080",
+    "secondaryColor": "#008000",
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "EU"
+    ],
+    "gameTypes": [
+        "Minigames",
+        "PvP"
+    ],
+    "website": "faramc.uk",
+    "socials": {
```

Comment:
I noticed you guys have a Twitter/X profile as well.
You can link it here if you want to